bugbot/rules/crash_small_volume.py

```diff
@@ -122,6 +122,11 @@ def handle_bug(self, bug, data):
             ),
             "keywords_to_remove": keywords_to_remove,
             "signatures": signatures,
+            "needinfo_ids": (
```

Comment:
I would use a more descriptive name like `needinfos_to_remove`.

Reply:
Done in b68ada9992ab73fcff6a13b394963f538041cdc9!